ui/App/TransactionCenter.svelte

```diff
@@ -15,9 +15,6 @@
 
 <style>
   .transaction-center {
-    position: fixed;
-    bottom: 2rem;
-    right: 2rem;
     z-index: 50;
```

Comment:
Same here, we can remove this because the footer takes care of it.

Reply:
oops, forgot that, my bad.